src/Tizen.Multimedia.MediaPlayer/Player/Player.Properties.cs

```diff
@@ -560,6 +560,81 @@ public bool ReplayGain
             }
         }
 
+        /// <summary>
+        /// Gets or sets the audio pitch.
+        /// </summary>
+        /// <value>The value indicating whether or not AudioPitch is enabled. The default is false.</value>
+        /// <remarks>This function is used for audio content only.</remarks>
+        /// <remarks>To set, the player must be in the <see cref="PlayerState.Idle"/> state.</remarks>
+        /// <exception cref="InvalidOperationException">The player is not in the valid state.</exception>
+        /// <exception cref="ObjectDisposedException">The player has already been disposed of.</exception>
+        /// <seealso cref="AudioPitch"/>
+        /// <since_tizen> 6 </since_tizen>
+        public bool AudioPitchEnabled
+        {
+            get
+            {
+                ValidateNotDisposed();
+                NativePlayer.IsAudioPitchEnabled(Handle, out var value).
+                    ThrowIfFailed(this, "Failed to get whether the audio pitch is enabled or not");
+                return value;
+            }
+
+            set
+            {
+                ValidateNotDisposed();
+                ValidatePlayerState(PlayerState.Idle);
+
+                NativePlayer.SetAudioPitchEnabled(Handle, value).
+                    ThrowIfFailed(this, "Failed to enable the audio pitch of the player");
+            }
+        }
+
+        /// <summary>
+        /// Gets or sets the audio pitch value.
+        /// </summary>
+        /// <value>The audio stream pitch value. The default is 1.</value>
+        /// <remarks>This function is used for audio content only.</remarks>
+        /// <exception cref="InvalidOperationException">A pitch is not enabled.</exception>
+        /// <exception cref="ObjectDisposedException">The player has already been disposed of.</exception>
+        /// <exception cref="ArgumentOutOfRangeException">
+        ///     value is less than 0.5.
+        ///     -or-<br/>
+        ///     value is greater than 2.0.<br/>
+        /// </exception>
+        /// <seealso cref="AudioPitchEnabled"/>
+        /// <since_tizen> 6 </since_tizen>
+        public float AudioPitch
+        {
+            get
+            {
+                ValidateNotDisposed();
+
+                if (AudioPitchEnabled == false)
+                    throw new InvalidOperationException("An audio pitch is not enabled.");
+
+                NativePlayer.GetAudioPitch(Handle, out var value).
+                    ThrowIfFailed(this, "Failed to get the audio pitch");
+
+                return value;
+            }
+
+            set
+            {
+                ValidateNotDisposed();
+
+                if (AudioPitchEnabled == false)
+                    throw new InvalidOperationException("An audio pitch is not enabled.");
```

Comment:
Is it need?
Usually developer pitch value change before enabling pitch feature
``` c#
player.AudioPitch = 1.1;
player.AudioPitchEnabled = true;
```

Reply:
AudioPItchEnabled has to be decided before the plugin is made.
so it would be called once when user create a player. on the other hand value can be changed continuously if AudioPitch is enabled.

Reply:
As described in `AudioPitchEnabled`, it have to set before prepare. It's because it have to be set before creating pipeline(Prepare state) in native.

Reply:
So, I don't understand, why need a check state of AudioPitchEnabled value in here?
User want to change AudioPitch value before enabling `AudioPitchEnabled`

Reply:
As I said before, enabling audio pitch value should be done before Prepare state.
(Before creating gstreamer pipeline in NATIVE framework)
So `AudioPitchEnabled` should be set first before set `AudioPitch`.

With your example above, native framework will return invalid operation error.

Reply:
Actually, if we don't check the value of `AudioPitchEnabled` in here, we get same error because native framework will return same error. But it have useless P/Invoke call.

Reply:
It is bad to API usability.
if possible, I want to fix native api behavior
Anyway, if impossible to change , I approve it

Reply:
I agree with your opinion in perspective of API usability, but it cannot be changed.